### DISCUSSION TITLE
Multiple styles on the same text

### DISCUSSION THREAD

**Discussion by 4l3m4oO** [friction2d/friction#51] [Ideas]
If i want to select just one letter or just one part of the text to change just that selection, it changes everything.

I think it would be nice to change only the part i selected and maybe manage some saved styles, to apply later on the project.

![image](https://github.com/friction2d/friction/assets/357488/6149bcbb-474b-482f-a2ac-5ce569e6dc70)

**Comment by rodlie** [comment]
Currently the input is plain text (not rich text), you can't set different style on individual characters/words etc.

This is of course something that should be improved. First we need to figure out how it's going to work, then find out if skia (the backend) supports rich text or markup (like pango), if not more work is needed, then at the end implement it.

Something to put on the roadmap :+1:
